app/views/posts/index.html.erb

```diff
@@ -2,15 +2,17 @@
 <html>
 <head>
   <title>All Posts</title>
+  <link rel="stylesheet" type="text/css" href="/assets/stylesheets/posts.css">
 </head>
 <body>
   <h1>All Posts</h1>
-  <ul>
+  <ul class="post-list">
     <% @posts.each do |post| %>
-      <li>
+      <li class="post-item">
         <h2><%= post.title %></h2>
```

Comment:
- [ ] The post title/body on the posts index page should be a link to the post's show page